addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -570,6 +570,15 @@ options.registry.gallery = options.Class.extend({
         this.$target.find('.carousel').attr('id', 'slideshow_' + uuid);
         this.$target.find('[data-target]').attr('data-target', '#slideshow_' + uuid);
     },
+    /**
+     * @override
+     */
+    onClone: function () {
+        var uuid = new Date().getTime();
+        this.$target.find('.carousel').attr('id', 'slideshow_' + uuid);
+        this.$target.find('[data-target]').attr('data-target', '#slideshow_' + uuid);
```

Comment:
Seems too generic as selector, `[data-target]` could be used by many other things in the future. Can't you find something more precise like you did with `[data-slide]` ?